pocs/focuser/focuser.py

```diff
@@ -486,14 +464,17 @@ def _autofocus(self,
             fitted = True
 
             # Guard against fitting failures, force best focus to stay within sweep range
-            if best_focus < focus_positions[0]:
-                self.logger.warning("Fitting failure: best focus {} below sweep limit {}".format(best_focus,
-                                                                                                 focus_positions[0]))
+            min_focus, max_focus = focus_positions[0], focus_positions[-1]
+            if best_focus < min_focus:
+                m = "Fitting failure: best focus {} below sweep limit {}".format(best_focus,
```

Comment:
Should this be `msg` instead of `m`?

Reply:
Probably, but then I'd have to split the string up in order to keep the line lengths below 100 characters.